zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -693,7 +693,7 @@ authentication.method.jb.name = JSON-based Authentication
 authentication.method.jb.popup.login.request = {0} : JSON-based Auth Login Request
 authentication.method.pb.field.label.postData				= Login Request POST Data (if any):
 authentication.method.pb.field.label.postDataRequired = Login Request POST Data *:
-authentication.method.pb.field.lable.loginPageUrl			= URL to GET Login Page:
+authentication.method.pb.field.label.loginPageUrl			= URL to GET Login Page:
```

Comment:
I've intentionally ignored all of the translations. If requested, I can apply commits to smooth out these changes.

Reply:
We’d have to ensure this is also fixed wherever it is used.

Reply:
Hopefully no add-on is using them too.